components/mixins/experience-event/experienceevent-marketing.schema.json

```diff
@@ -20,6 +23,21 @@
           "title": "Marketing",
           "$ref": "https://ns.adobe.com/xdm/context/marketing",
           "description": "The information related to marketing activities that are active with the touchpoint."
+        },
+        "xdm:internalMarketing": {
+          "title": "Internal Marketing",
+          "$ref": "https://ns.adobe.com/xdm/context/marketing",
+          "description": "Internal Marketing data type that contains group, name, and code."
+        },
+        "xdm:productFindingMethod": {
```

Comment:
If this property is supposed to be at the root of the Marketing mixin, can this property be part of the marketing data type ?

Reply:
Looks like this field "internal marketing" is based on the datatype "marketing". I guess we could have them defined directly under the mixin rather than including it under marketing datatype.  marketing itself is a another field in this mixin. So to summarize both fields "marketing" and "internal marketing" share the same datatype.

Reply:
@mjasrotia @prabhum2
Let's move this to a V2 change and just approve the meta:tags for now?